README.md

```diff
@@ -1,9 +1,55 @@
 [![Build Status](https://travis-ci.org/TREP-LABS/realdrip-backend.svg?branch=master)](https://travis-ci.org/TREP-LABS/realdrip-backend)
 [![Coverage Status](https://coveralls.io/repos/github/TREP-LABS/realdrip-backend/badge.svg?branch=master)](https://coveralls.io/github/TREP-LABS/realdrip-backend?branch=master)
-## RealDrip Backend
-`A node-js server using express-js
-`
+
+![GitHub contributors](https://img.shields.io/github/contributors/trep-labs/realdrip-backend)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/m/trep-labs/realdrip-backend)
+![GitHub pull requests](https://img.shields.io/github/issues-pr/trep-labs/realdrip-backend)
+![GitHub closed pull requests](https://img.shields.io/github/issues-pr-closed-raw/trep-labs/realdrip-backend)
+# RealDrip Backend
+ > [Learn about The Realdrip Device](https://treplabs.co/realdrip)
+
 #### The Setup integrated the following CI/CD tools
  - TravisCI
  - Coveralls.io
- - Jest
\ No newline at end of file
+ - Jest & Suppertest
```

Comment:
Jest and Supertest are not CI/CD tools, please remove them.

Reply:
okay